core/unit_test/TestGraph.hpp

```diff
@@ -210,6 +208,38 @@ TEST_F(TEST_CATEGORY_FIXTURE(graph),
   ASSERT_EQ(0, bugs_host());
```

Comment:
Looks like you missed an occurrence
https://github.com/kokkos/kokkos/blob/a998687f9f1192985546333aeeafd5f5b600c915/core/unit_test/TestGraph.hpp#L204-L208

Reply:
Indeed !